ci/docker/runtime_functions.sh

```diff
@@ -1296,6 +1296,20 @@ nightly_scala_demo_test_cpu() {
     bash bin/run_im.sh
 }
 
+nightly_estimator_cnn_gpu() {
+    set -ex
+    cd /work/mxnet/tests/nightly/estimator
+    export PYTHONPATH=/work/mxnet/python/
+    python test_estimator_cnn.py --type gpu
```

Comment:
what's the reason to run a python script instead of using nosetest and assert accuracy at the end?

Reply:
Since we are executing a single test here, running the python script suffices. We can shift to nosetests if we add more tests in the future.